tools/src/test/java/org/apache/kafka/tools/consumer/group/DescribeConsumerGroupTest.java

```diff
@@ -69,35 +91,60 @@ public class DescribeConsumerGroupTest extends ConsumerGroupCommandTest {
         DESCRIBE_TYPES = describeTypes;
     }
 
-    @ParameterizedTest(name = TEST_WITH_PARAMETERIZED_QUORUM_AND_GROUP_PROTOCOL_NAMES)
-    @MethodSource({"getTestQuorumAndGroupProtocolParametersAll"})
-    public void testDescribeNonExistingGroup(String quorum, String groupProtocol) {
-        createOffsetsTopic(listenerName(), new Properties());
+    DescribeConsumerGroupTest(ClusterInstance clusterInstance) {
+        this.clusterInstance = clusterInstance;
+    }
+
+    private static List<ClusterConfig> generator() {
```

Comment:
reuse `return ConsumerGroupCommandTestUtils.generator();`?

Reply:
In this test, we only run 3 servers, not like default `ConsumerGroupCommandTestUtils.generator()` running 5 servers. This is to address https://github.com/apache/kafka/pull/15908#issuecomment-2131274897. Do we want to change it back?

Reply:
> ConsumerGroupCommandTestUtils.generator() running 5 servers.

that is updated to run 3 clusters now. see https://github.com/apache/kafka/blob/trunk/tools/src/test/java/org/apache/kafka/tools/consumer/group/ConsumerGroupCommandTestUtils.java#L83

Reply:
Sorry, I didn't double check for it. Yeah, update to `ConsumerGroupCommandTestUtils.generator();`. Thanks.